src/config.js

```diff
@@ -69,7 +69,7 @@ export class Config {
 
     // Base url is string. Configure.
     newClient.configure(configure => {
-      configure.withBaseUrl(configureMethod);
+      configure.withBaseUrl(String(configureMethod));
```

Comment:
I'd rather not cast to string, but check if it's a string and throw an error otherwise.